CONTRIBUTING.md

```diff
@@ -111,6 +170,15 @@ crates/meroctl/src/cli/app/list.rs
 
 ### Error Handling
 
+- We use the eyre crate extensively in our code, and import it as follows:
+  
+  ```rust
+  use eyre::{Result as EyreResult};
+  ```
+
+- Almost no unwrapping (acceptable in tests and possibly when dealing with thread join handlers).
```

Comment:
any joining should be a handle-able issue, it's only acceptable when it's absolutely certain it won't fail, like integration's `ReprTransmute` where we have a bunch of `.expect("infallible conversion")`